app/models/asso_relation.rb

```diff
@@ -0,0 +1,15 @@
+class AssoRelation < ActiveRecord::Base
+  belongs_to :factory
+  belongs_to :asso, foreign_key: "asso_id", class_name: 'Factory'
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.